.github/workflows/update-charts.yml

```diff
@@ -0,0 +1,126 @@
+name: Update helm charts
+
+on:
+  repository_dispatch:
+    types: [update-chart]
+
+jobs:
+  check-update-type:
+    name: Detect update type
+    runs-on: ubuntu-latest
+    outputs:
+      update_type: ${{ steps.step1.outputs.update_type }}
+    steps:
+      - name: Check if it is patch update
+        id: step1
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let oldVersion = context.payload.client_payload.oldVersion.split(".")
+            let version = context.payload.client_payload.version.split(".")
+            if (oldVersion.length != 3 && version.length != 3) {
+                    core.setFailed("Invalid version format")
```

Comment:
goalpost moving, but I suppose this breaks for rcs.

Reply:
I think this should still work also for RCs. We are breaking up the version by `.`, hence `1.0.0-rc1` would be broken into `["1", "0", "0-rc1"]` which should be fine.

Maybe I didn't get it :sweat_smile:

Reply:
no, you are right, it should still work :). Resolving.